### PR TITLE
x52, here it comes

### PR DESCRIPTION
**Pull request by Delnth**
Add the x52 base and all of its components, assets and the like. Lighting and jobs don't work yet. Until I can address these issues, do not merge.

:cl: Koenigsegg
maptweak: Add x52 base to geminus sewers (With doors restricted to x52 personnel)
maptweak: Add virology, R&D, Hydroponics, chemistry, surgery and integrated circuits to the base as well as a storage room with a few items of interest.
rscadd: Add X52 Researcher and Research Director
/:cl:

### CODE REVIEW

**Comment by BDpuffy420** [comment]
Duplicated APCs in area

**Review by bloxgate — COMMENTED** [review]
Some of those `rscadd`s in your changelog should probably be changed to `maptweak`

**Comment by Delnth** [comment]
Duplicate APC? There's only one in the area that I have coded in. I shoudn't need to make a new instance of said APC, since they update at round start.

As for the changelog, I'll change it.

**Comment by BDpuffy420** [comment]
That’s what’s the Travis log reading, it’s an APC underground

**Comment by Delnth** [comment]
I'm fixing the APC problem. In the meantime, any idea why the jobs do not appear in the ingame list? Everything is setup correctly.

**Comment by BDpuffy420** [comment]
Is the job in the allow_spawn list?

**Comment by Delnth** [comment]
Okay now I am asking a developper to look into the problem because it just seem like Travis is picking up a false alarm. Nothing's wrong with the APC. It is tested and compiled without a problem, yet some random APC is having conflicts with the X52 one. 

Again, there is absolutely **nothing wrong**, yet it refuses to pass Travis checks for some reason. I am not going to attempt fixing the issue until I get some assistance.

**Comment by Delnth** [comment]
Last commit fixes the X52 researchers faction, so they can actually open the fakewall.
The PR is ready to go.

File: maps/geminus_city/x52_jobs.dm [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/datum/job/geminus_x52/researcher
+	title = "X52 Researcher"
+	spawn_faction = "Insurrection"
```
Comment (caelaislinn) [inline maps/geminus_city/x52_jobs.dm]: Can you change spawn_faction to "X52"

File: maps/geminus_city/x52_jobs.dm [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	access = list(access_x52)
+	selection_color = " #a01b01"
+	spawnpoint_override = "X52 Researcher"
+	alt_titles = null
+	whitelisted_species = list(/datum/species/human)
+
+
+/datum/job/geminus_x52/research_director
+	title = "X52 Research Director"
+	spawn_faction = "Insurrection"
```
Comment (caelaislinn) [inline maps/geminus_city/x52_jobs.dm]: And here change spawn_faction to "X52"

File: maps/geminus_city/x52_outfits.dm [hunk 3]
```diff
@@ -0,0 +1,47 @@
+/decl/hierarchy/outfit/job/geminus_x52
+	name = "X52 Researcher"
+
+	head = null
+	glasses = null
+	mask = null
+	suit = null
+	uniform = /obj/item/clothing/under/sterile
+	shoes = /obj/item/clothing/shoes/white
+	l_ear = /obj/item/device/radio/headset/insurrection
```
Comment (caelaislinn) [inline maps/geminus_city/x52_outfits.dm]: Don't give them innie comms, just an ordinary colonist headset for now. I'll handle their comms separately

File: maps/geminus_city/x52_outfits.dm [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+	flags = 0
+
+/decl/hierarchy/outfit/job/geminus_x52/equip_id(mob/living/carbon/human/H, rank, assignment)
+	. = ..()
+
+	var/obj/item/weapon/card/id/W = .
+	if(W)
+		W.rank = "Colonist"
+		W.assignment = "Colonist"
+		W.update_name()
```
Comment (caelaislinn) [inline maps/geminus_city/x52_outfits.dm]: Change this to 
```
/decl/hierarchy/outfit/job/colonist/equip_id(mob/living/carbon/human/H)
	var/obj/item/weapon/card/id/C = ..()
	C.assignment = "Colonist"
	H.set_id_info(C)
```
to match the innie IDs

File: maps/geminus_city/x52_outfits.dm [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+	flags = 0
+
+/decl/hierarchy/outfit/job/geminus_x52_rd/equip_id(mob/living/carbon/human/H, rank, assignment)
+	. = ..()
+
+	var/obj/item/weapon/card/id/W = .
+	if(W)
+		W.rank = "Colonist"
+		W.assignment = "Colonist"
+		W.update_name()
```
Comment (caelaislinn) [inline maps/geminus_city/x52_outfits.dm]: Change this equip_id() same as above

**Review by caelaislinn — CHANGES_REQUESTED** [review]
One other change, go into the map and change var/faction_locked for the fakewalls to "X52". 

Although they're a subfaction of innies, my intent is for them to be mostly separate with separate objectives, and not sharing any access or comms (at least at start).

File: maps/geminus_city/x52_jobs.dm [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+	latejoin_at_spawnpoints = 1
+	total_positions = 4
+	spawn_positions = 4
+	outfit_type = /decl/hierarchy/outfit/job/geminus_x52
+	access = list(access_x52)
+	selection_color = " #a01b01"
+	spawnpoint_override = "X52 Researcher"
+	alt_titles = null
+	whitelisted_species = list(/datum/species/human)
```
Comment (caelaislinn) [inline maps/geminus_city/x52_jobs.dm:12]: Can you add 
```
intro_blurb = "You are X-52! Working for a top secret military research project, one day you decided to go AWOL with your secrets and expertise. Now you're out for yourselves. Just remember the UNSC will always be looking for you..."
```

File: maps/geminus_city/x52_jobs.dm [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+	department_flag = COM
+	latejoin_at_spawnpoints = 1
+	total_positions = 1
+	spawn_positions = 1
+	outfit_type = /decl/hierarchy/outfit/job/geminus_x52_rd
+	access = list(access_x52_rd, access_x52)
+	selection_color = " #a01b01"
+	spawnpoint_override = "X52 RD"
+	alt_titles = null
+	whitelisted_species = list(/datum/species/human)
```
Comment (caelaislinn) [inline maps/geminus_city/x52_jobs.dm:28]: And add
```
intro_blurb = "You are X-52! Working for a top secret military research project, one day you decided to go AWOL with your secrets and expertise. Now you're out for yourselves. Just remember the UNSC will always be looking for you..."
```

**Review by caelaislinn — CHANGES_REQUESTED** [review]
Can you go to /datum/job/geminus_innie/officer in maps\geminus_city\innie_jobs.dm and add
```
	open_slot_on_death = 1
	track_players = 1
```
This will ensure that officers can respawn, as well as basic innies

**Comment by bloxgate** [comment]
Travis failure is a false alarm due to #1732. I'll restart that specific test, but otherwise this is good to go once Cael is happy with it.

**Review by caelaislinn — APPROVED** [review]

**Comment by caelaislinn** [comment]
Happy to merge. This is content for live testing and will probably go through some dramatic changes over the next week.

**Comment by Delnth** [comment]
I might as well move the base elsewhere too. I'm afraid the proximity from the sewer hallways might be an easy lead for anyone willing to meta its location. We will see how it goes.

**Pull request merged**